client/webpack.rails.config.js

```diff
@@ -7,18 +7,19 @@
 var config = require("./webpack.common.config");
 
 config.entry.push("./scripts/rails_only"); // rails specific assets
-config.output = { filename: "webpack-bundle.js",
+config.output = { filename: "client-bundle.js",
                   path: "../app/assets/javascripts" };
 config.externals = { jquery: "var jQuery" }; // load jQuery from cdn or rails asset pipeline
 config.module.loaders.push(
-  { test: /\.jsx$/, loaders: ["es6", "jsx?harmony"] },
-  // Next 2 lines expose jQuery and $ to any JavaScript files loaded after webpack-bundle.js
+  { test: /\.jsx$/, exclude: /node_modules/, loader: 'babel-loader'},
+  // Next 2 lines expose jQuery and $ to any JavaScript files loaded after client-bundle.js
```

Comment:
I believe you might want some .jsx from node_modules to be included? For example, I use the react-paginate library.